### ISSUE TITLE
Could not initialize project using Firebeetle-2 from DF Robot.

### ISSUE THREAD

**Issue by markAustin902** [platformio/platformio-home#6587]
I get the following error when trying to create a new project when I select the board type as "DFRobot Firebeetle 2 ESP32-E"

PIO Core Call Error: "Resolving dfrobot_firebeetle2_esp32e dependencies...\r\n\n\nUnknownBoard: Unknown board ID 'dfrobot_firebeetle2_esp32e'"

**Comment by lappiesja** [comment]
Hi, were you able to resolve this issue?

**Comment by markAustin902** [comment]
Yes, just use the 'Firebeetle-ESP32 (DFRobot)' Board type.  This has been working OK for my applications with the Firebeetle1 and Firebeetle2 fitted with ESP32-WROOM-32E / D.

<img width="542" height="58" alt="Image" src="https://github.com/user-attachments/assets/ff6ffee4-4474-486e-b0a2-ee3f4f1a9a5a" />

In the platformio.ini is will be added as Firebeetle32

[env:firebeetle32]
platform = espressif32
board = firebeetle32
framework = arduino
